.github/workflows/release.yml

```diff
@@ -185,7 +185,7 @@ jobs:
           JIRA_DESCRIPTION: ${{ env.jira_description }}
```

Comment:
1. The changes made to the `RELEASE_NOTES` variable seem to have removed the line breaks and added some additional formatting.
2. The use of multiple `sed` commands on the `RELEASE_NOTES` variable could potentially make the code harder to read and maintain.
3. Consider breaking down the string manipulation into smaller, more manageable steps for better readability.
4. Verify that the changes made do not affect the functionality of the script or introduce any potential errors.